### PR TITLE
update for Subnet Configuration Generator v1 release

### PR DESCRIPTION
**Pull request by wanwiset25**

### CODE REVIEW

File: backend/src/server.ts [hunk 1]
```diff
@@ -9,9 +9,9 @@ const app = new App([new Route()]);
 const server = http.createServer(app.getServer());
 const eventHandler = new EventsHandler(server);
 
-server.listen(3000, () => {
+server.listen(5213, () => {
```
Comment (benjamin202410) [inline backend/src/server.ts:12]: can you make environment for this port?

File: backend/src/services/block.service.ts [hunk 2]
```diff
@@ -211,10 +213,15 @@ export class BlockService {
...
     const { timestamp } = await this.parentChainClient.getParentChainBlockBySubnetHash(smartContractCommittedHash);
+    const { number: subnetCommittedNumber } = await this.subnetClient.getLatestCommittedBlockInfo();
 
-    const timeDiff = new Date().getTime() / 1000 - parseInt(timestamp.toString());
-
-    const isProcessing = (mode == 'full' && timeDiff < 120) || (mode == 'lite' && timeDiff < 1000);
+    let isProcessing = true;
+    const blockDiff = subnetCommittedNumber - smartContractHeight;
+    if (mode == 'lite') {
+      if (blockDiff > 1000) isProcessing = false;
```
Comment (benjamin202410) [inline backend/src/services/block.service.ts:221]: always {}

Comment (benjamin202410) [reply backend/src/services/block.service.ts:221]: or use : true ? false

Comment (benjamin202410) [reply backend/src/services/block.service.ts:221]: or have 2 condition together

**Review by GalaxySciTech — APPROVED** [review]

**Pull request merged**